TASK-5/Utils/BogusRepository.cs

```diff
@@ -0,0 +1,28 @@
+using Bogus;
+
+namespace Task5.Utils;
+
+public class BogusRepository
+{
+    public IEnumerable<Ware> GetWares(List<Ware> listOfWares)
+    {
+        var wareGenerator = new Faker<Ware>()
+            .RuleFor(w => w.Id, (f, w) => f.Commerce.Ean8())
+            .RuleFor(w => w.Name, (f, w) => f.Commerce.ProductName())
+            .RuleFor(w => w.Category, (f, w) => f.Commerce.Categories(1)[0])
+            .RuleFor(w => w.Price, (f, w) => f.Commerce.Price());
+        
+        return wareGenerator.Generate(listOfWares.Count);
+    }
+
+    public IEnumerable<Customer> GetCustomers(List<Customer> listOfCustomers)
+    {
+        var customerGenerator = new Faker<Customer>()
+            .RuleFor(c => c.PassportId, (f, c) => f.Finance.Account()) 
+            .RuleFor(c => c.Name, (f, c) => f.Name.FullName())
+            .RuleFor(c => c.Age, (f, c) => f.Person.Random.Int(6, 110))
```

Comment:
Change hardcode for random as age variables

Reply:
Created fields in class Customer:
  public const int MinimumAge = 4;
  public const int MaximumAge = 130;